docs/source/operations/projections/webmerc.rst

```diff
@@ -21,21 +20,21 @@ From `Wikipedia <https://en.wikipedia.org/wiki/Web_Mercator>`_:
     historically.
 
 
-+---------------------+----------------------------------------------------------+
-| **Classification**  | Cylindrical (non conformant if used with ellipsoid)      |
-+---------------------+----------------------------------------------------------+
-| **Available forms** | Forward and inverse, spherical projection                |
-+---------------------+----------------------------------------------------------+
-| **Defined area**    | Global, but best used near the equator                   |
-+---------------------+----------------------------------------------------------+
-| **Alias**           | webmerc                                                  |
-+---------------------+----------------------------------------------------------+
-| **Domain**          | 2D                                                       |
-+---------------------+----------------------------------------------------------+
-| **Input type**      | Geodetic coordinates                                     |
-+---------------------+----------------------------------------------------------+
-| **Output type**     | Projected coordinates                                    |
-+---------------------+----------------------------------------------------------+
++---------------------+-----------------------------------------------------------------+
+| **Classification**  | Non-conformal cylindrical projection of an WGS 84 ellipsoid     |
++---------------------+-----------------------------------------------------------------+
+| **Available forms** | Forward and inverse, ellipsoidal projection                     |
++---------------------+-----------------------------------------------------------------+
+| **Defined area**    | Global, low linear distortions near the equator                 |
```

Comment:
I think it is better to just say that it is globally defined and not go into where distortions are at a minimum. The original text is just a copy of what's written for Mercator which is kind of not valid here anyway.